api/test/unit/views/media_views_test.py

```diff
@@ -241,3 +243,19 @@ def test_thumb_full_size(api_client, media_type, media_factory, requests):
         assert (
             entry in requests.sent_requests[1].request.url
         ), f"{entry} not found in prepared request url: {requests.sent_requests[0].request.url}"
+
+
+@pytest.mark.parametrize("count", [1, 3])
+def test_thumb_timeout(count: int):
+    cache.clear()  # to prevent effect of other runs
```

Comment:
This will clear the cache for every other test... which in theory should be fine but also could not be (in particular with the "integration" tests). Can we use `RedisMock` here like we do in the other unit tests, rather than relying on the real elasticsearch? That way we get a fresh redis to play with each test run. See this test fixture in `validate_images_test` for an example:

https://github.com/WordPress/openverse-api/blob/f0c44d571249a2631eef5a13ea0d56702210daf3/api/test/unit/utils/validate_images_test.py#L12-L22

Reply:
Added a fake Redis fixture in 3b9eca4.